raft/src/test/java/org/apache/kafka/raft/RaftClientTestContext.java

```diff
@@ -576,6 +577,10 @@ void assertSentDescribeQuorumResponse(
             partitionData,
             nodes
         );
+
+        List<ReplicaState> sortedVoters = response.topics().get(0).partitions().get(0).currentVoters().stream().sorted(Comparator.comparingInt(ReplicaState::replicaId)).collect(Collectors.toList());
+        response.topics().get(0).partitions().get(0).setCurrentVoters(sortedVoters);
```

Comment:
Let try to split this over a few lines. I am okay with this formatting:
```java
        List<ReplicaState> sortedVoters = response
            .topics()
            .get(0)
            .partitions()
            .get(0)
            .currentVoters()
            .stream()
            .sorted(Comparator.comparingInt(ReplicaState::replicaId))
            .collect(Collectors.toList());
        response.topics().get(0).partitions().get(0).setCurrentVoters(sortedVoters);
```

Reply:
thanks for the feedback, updated in latest commit